modules/hivemq-edge-module-plc4x/src/main/java/com/hivemq/edge/adapters/plc4x/types/eip/EIPAdapterConfig.java

```diff
@@ -19,81 +19,86 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import com.hivemq.edge.adapters.plc4x.model.Plc4xAdapterConfig;
-import com.hivemq.edge.adapters.plc4x.types.ab.ABAdapterConfig;
 import com.hivemq.edge.modules.adapters.annotations.ModuleConfigField;
 import com.hivemq.extension.sdk.api.annotations.NotNull;
 
-import java.math.BigInteger;
-import java.time.Duration;
-import java.time.LocalDate;
-import java.time.LocalDateTime;
-import java.time.LocalTime;
 import java.util.ArrayList;
 import java.util.List;
 
 
 public class EIPAdapterConfig extends Plc4xAdapterConfig {
 
+    @JsonProperty("port")
```

Comment:
Just curious: Wasnt a port needed before?

Reply:
The parent class already contains a port, just overridden to add a default value here.